robottelo/api/utils.py

```diff
@@ -25,6 +25,16 @@
 from robottelo.decorators import bz_bug_is_open
 
 
+def repository_synchronize_with_timeout(repo_id, timeout=300):
```

Comment:
I would suggest to make that method more generic as we see similar code in many different modules. Not sure how to implement that, but it will be good to receive command statement like `entities.Repository(id=repo_id).sync()` as input parameter